src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -128,6 +129,8 @@ class instance_naxxramas : public InstanceMapScript
         uint64 _thaddiusPortalGUID;
 
         // NPCs
+        std::list<uint64> PatchwerkRoomTrash;
```

Comment:
Is there a reason for list over vector? You're always pushing to the back.

Reply:
was writing it @ 4am, will rewrite it to a vector, thanks